keymanager/src/policy/signers.rs

```diff
@@ -18,6 +18,38 @@ pub struct TrustedSigners {
     pub threshold: u64,
 }
 
+#[cfg(feature = "debug-mock-sgx")]
+impl TrustedSigners {
+    /// An UNSAFE set of trusted signers using well-known debug keys.
+    pub fn mock() -> Self {
```

Comment:
```suggestion
    pub fn unsafe_mock() -> Self {
```
If you want to emphasize unsafe.

Reply:
Changed.